apps/ejabberd/src/mod_mam_muc.erl

```diff
@@ -331,7 +337,12 @@ is_user_identity_hidden(From, ArcJID) ->
 
 -spec can_access_room(From :: ejabberd:jid(), To :: ejabberd:jid()) -> boolean().
 can_access_room(From, To) ->
-    ejabberd_hooks:run_fold(can_access_room, To#jid.lserver, false, [From, To]).
+    Stanza = mongoose_stanza:new(), % some day original stanza will reach this stage
```

Comment:
Please take a look from where function`can_access_room` is called. This hook is private to `mod_mam_muc` that's why I don't think there will be a need to pass the whole accumulator to it. If at some point we figure out it's worth adding, then it will be changed. I'd limit number of changes just for now.